test/functional/wallet_labels.py

```diff
@@ -92,17 +92,25 @@ def _run_subtest(self, accounts_api, node):
         # recognize the label/address associations.
         labels = [Label(name, accounts_api) for name in ("a", "b", "c", "d", "e")]
         for label in labels:
-            label.add_receive_address(node.getlabeladdress(label=label.name, force=True))
+            if accounts_api:
+                address = node.getaccountaddress(label.name)
+            else:
+                address = node.getnewaddress()
```

Comment:
`address = node.getnewaddress(label.name)`?
(or is the point here to test setlabel?)

Reply:
No, yours is better. `setlabel` is tested further down.

Fixed and pushed the new version.